inference-engine/src/mkldnn_plugin/nodes/psroi.cpp

```diff
@@ -94,81 +111,79 @@ class PSROIPoolingImpl: public ExtLayerBase {
             float roi_width   = 0.0f;
             float roi_height  = 0.0f;
 
-            if (mode_ == "bilinear") {
-                roi_start_w = bottom_rois[1] * spatial_scale_;
-                roi_start_h = bottom_rois[2] * spatial_scale_;
-                roi_end_w = bottom_rois[3] * spatial_scale_;
-                roi_end_h = bottom_rois[4] * spatial_scale_;
-                roi_width  = roi_end_w - roi_start_w;
-                roi_height = roi_end_h - roi_start_h;
-            } else if (mode_ == "average") {
+            size_t index = n*nc*nh*nw;
```

Comment:
Looks like such index computation is not always valid. What if nc % block_size !=0? In fact for this situation channels are implicitly rounded up to satisfy nc % block_size ==0, so physical memory need to store blocked blob is increased (at the same time inDims[1]/outDims[1] stays unaligned).
General recommendation: use BlockingDesc since it respects channels alignment.

Reply:
We intentionally supports only planar output. Anyway, it will be completed in the special PR. Now psroi rollbacked.